18F/dol-whd-14c
# Two Factor Authentication

#### What's this PR do?
- Created solution to support Two Factor Authentication (2FA) through email. 
- Added and new web API to support resend verification code. 
- Updated login page to support verification code.
#### How should this be manually tested?
Ensure Two Factor Authentication is enable in the database. Log in to 14c Application; user is prompted for a valid verification code. User should be able to log in to the application use the security code from email sent by the 14c application.
#### What are the relevant tickets?
#418 - UI Implementation - 2FA page for login workflow 
#203 - As a user I would like application to support 2FA to feel comfortable about security hardness of the system
